core/Array.savi

```diff
@@ -65,8 +65,8 @@
     @val_from_cpointer(ptr, size, space)
 
   :fun ref trim_in_place(from USize, to USize = -1)
-    to = to.min(@_size)
-    from = from.min(to)
+    to = to.at_most(@_size)
+    from = from.at_most(to)
```

Comment:
This is probably my least favorite example for `at_most` and the like. In some sense `min`/`max` benefited from being more mathematical and less english, because here we're combining units (indexes vs. length) and using simple words for the variables.

But maybe it's just that this operation is less clear, it's not really numbers at all, but "locations" so to speak

Reply:
Yeah I still see it as an improvement, personally - even in this case.